src/Elastic.Apm/Config/ConfigConsts.cs

```diff
@@ -129,33 +136,37 @@ public static class EnvVarNames
 			public const string TransactionSampleRate = Prefix + "TRANSACTION_SAMPLE_RATE";
 			public const string UseElasticTraceparentHeader = Prefix + "USE_ELASTIC_TRACEPARENT_HEADER";
 			public const string VerifyServerCert = Prefix + "VERIFY_SERVER_CERT";
-			public const string ExcludedNamespaces = Prefix + "EXCLUDED_NAMESPACES";
-			public const string ApplicationNamespaces = Prefix + "APPLICATION_NAMESPACES";
 			public static string TransactionIgnoreUrls = Prefix + "TRANSACTION_IGNORE_URLS";
-			//This setting is Full Framework only:
-			public const string FullFrameworkConfigurationReaderType = Prefix + "FULL_FRAMEWORK_CONFIGURATION_READER_TYPE";
+			public static string Recording = Prefix + "RECORDING";
 		}
 
 		public static class KeyNames
 		{
-			private const string Prefix = "ElasticApm:";
+			public const string ApiKey = Prefix + nameof(ApiKey);
+			public const string ApplicationNamespaces = Prefix + nameof(ApplicationNamespaces);
 
 			public const string CaptureBody = Prefix + nameof(CaptureBody);
 			public const string CaptureBodyContentTypes = Prefix + nameof(CaptureBodyContentTypes);
 			public const string CaptureHeaders = Prefix + nameof(CaptureHeaders);
 			public const string CentralConfig = Prefix + nameof(CentralConfig);
 			public const string DisableMetrics = Prefix + nameof(DisableMetrics);
+			public const string Enabled = Prefix + nameof(Enabled);
 			public const string Environment = Prefix + nameof(Environment);
+			public const string ExcludedNamespaces = Prefix + nameof(ExcludedNamespaces);
+
 			public const string FlushInterval = Prefix + nameof(FlushInterval);
+
+			//This setting is Full Framework only:
+			public const string FullFrameworkConfigurationReaderType = Prefix + nameof(FullFrameworkConfigurationReaderType);
 			public const string GlobalLabels = Prefix + nameof(GlobalLabels);
 			public const string HostName = Prefix + nameof(HostName);
 			public const string LogLevel = Prefix + nameof(LogLevel);
 			public const string MaxBatchEventCount = Prefix + nameof(MaxBatchEventCount);
 			public const string MaxQueueEventCount = Prefix + nameof(MaxQueueEventCount);
 			public const string MetricsInterval = Prefix + nameof(MetricsInterval);
+			private const string Prefix = "ElasticApm:";
```

Comment:
super minor nitpick: looks like ordering is alphabetical, but I think it should be accessibility -> alphabetical, so that `private` members appear before `public`

Reply:
Oh - no idea why mi Rider did this. Fixed it.